### PR TITLE
Core/3 add mailgun and mailchimp

### PR DESCRIPTION
**Pull request by Enrique199413**
Add the mailchimp function to integrate with neopraxis

### CODE REVIEW

File: routes/api/mailchimp.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+				'email_address': req.body.email,
+				'status': 'subscribed',
+			})
+			.end(function (err, response) {
+				if ((response.status === 400 && response.body.title === "Invalid Resource")) {
+					res.status(412).send({ error: response.body.detail });
+				} else if ((response.status === 400 && response.body.title === "Member Exists")) {
+					res.status(403).send({ error: 200 });
+				} else {
+					res.status(200).send({ suscribe: true });
```
Comment (AlanSanchezP) [inline routes/api/mailchimp.js]: Change suscribe to subscribe.

File: templates/views/layouts/default.html [hunk 2]
```diff
@@ -208,7 +208,10 @@
 							</a>
 						</div>
 						<div class="newsletter-box">
-							<input type="text" name="" value="" placeholder="newsletter" class="newsletter-input neopraxis-font-myriadpro-light uppercase letter-spacing-dot-six"  onfocus="this.placeholder = ''" onblur="this.placeholder = 'newsletter'">
+							<form id="newsletterInput">
+								<input type="text" name="email" value="" placeholder="newsletter" class="newsletter-input neopraxis-font-myriadpro-light uppercase letter-spacing-dot-six"  onfocus="this.placeholder = ''" onblur="this.placeholder = 'newsletter'" id="inputText">
```
Comment (AlanSanchezP) [inline templates/views/layouts/default.html]: It'd be better to move all js logic into separated files.

Comment (Enrique199413) [inline templates/views/layouts/default.html]: Consider refactor. For this time I added this @AlanSanchezP .

File: routes/api/mailchimp.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
+/**
+ * Created by enriquelc on 17/05/17.
+ */
+var request = require('superagent');
+var mailchimpInstance, listUniqueId, mailchimpApiKey;
+
+exports.newsletter = function(req, res) {
+
+	if (!process.env.MAILCHIMP_INSTANCE || !process.env.MAILCHIMP_UNIQUE_LIST_ID || !process.env.MAILCHIMP_API_KEY) {
+		console.log('Unable to integrate mailchimp - no mailchimp credentials provided');
```
Comment (jarodriguez80) [inline routes/api/mailchimp.js]: We can use a logger instead console.log ?

**Pull request merged**